docs/pages/installation.mdx

```diff
@@ -71,11 +71,10 @@ information on obtaining Teleport binaries compatible with Teleport Cloud.
 
 <Tabs>
 <TabItem scope={["oss"]} label="Open Source">
-We provide pre-built Docker images for every version of Teleport.
+We provide pre-built `amd64`, `arm`, and `arm64` Docker images for every version of Teleport.
 
-These images are hosted on Amazon ECR Public. All tags under
-`public.ecr.aws/gravitational/teleport`
-[are Teleport Open Source images](https://gallery.ecr.aws/gravitational/teleport).
+These images are hosted on Amazon ECR Public. All tags under `public.ecr.aws/gravitational/teleport` [are Teleport Open Source images](https://gallery.ecr.aws/gravitational/teleport).
+You can specify the architecture of a Teleport Docker image by adding a postfix to the image tag, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`. Versions without an architecture postfix use the `amd64` architecture, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)`. 
```

Comment:
Should we have a partial file for the OSS Docker images the way we do for enterprise?